inc/template-tags.php

```diff
@@ -181,9 +186,8 @@ function twentysixteen_post_thumbnail() {
 function twentysixteen_excerpt_more( $more ) {
 	$link = sprintf( '<a href="%1$s" class="more-link">%2$s</a>',
 		esc_url( get_permalink( get_the_ID() ) ),
-		/* translators: %s: Name of current post */
-		sprintf( __( 'Continue reading %s', 'twentysixteen' ), '<span class="screen-reader-text">' . esc_html( get_the_title( get_the_ID() ) ) . '</span>' )
-		);
+		sprintf( _x( 'Continue reading %s', 'Name of current post', 'twentysixteen' ), '<span class="screen-reader-text">' . esc_html( get_the_title( get_the_ID() ) ) . '</span>' )
```

Comment:
I have not used Poedit recently but the developer told me he going to add support for it. GlotPress has always supported it. You can see it showing up in the German translation: https://translate.wordpress.org/projects/wp-themes/twentysixteen/de/default?filters%5Bterm%5D=Continue+reading+%25s&filters%5Buser_login%5D=&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filter=Filter&sort%5Bby%5D=priority&sort%5Bhow%5D=desc